pkg/asset/agent/image/ignition.go

```diff
@@ -163,14 +163,27 @@ func (a *Ignition) Generate(dependencies asset.Parents) error {
 		return err
 	}
 
-	// Use bootstrap script to get container images
-	err = bootstrap.AddStorageFiles(&config, "/usr/local/bin/release-image.sh",
-		"bootstrap/files/usr/local/bin/release-image.sh.template",
-		struct{ ReleaseImage string }{ReleaseImage: agentManifests.ClusterImageSet.Spec.ReleaseImage})
-	if err != nil {
+	// Set up bootstrap service recording
```

Comment:
I was trying to understand why bootstrap-service-record.sh is being pulled in to the ignition. I see that release-image-download.sh uses it.